cmd/podman/system/service.go

```diff
@@ -130,7 +130,7 @@ func resolveAPIURI(uri []string) (string, error) {
 		logrus.Info("Using systemd socket activation to determine API endpoint")
 		return "", nil
 	case rootless.IsRootless():
-		xdg, err := util.GetRuntimeDir()
+		xdg, err := util.GetRootlessRuntimeDir()
```

Comment:
Depending on how consolidation of `IsRootless` goes, this might eventually be able to call the c/storage version directly.